### DISCUSSION TITLE
Cannot generate schema

### DISCUSSION THREAD

**Discussion by Overload119** [planetscale/discussion#102] [Bug Reports] [answered]
![CleanShot 2021-09-21 at 17 23 25](https://user-images.githubusercontent.com/1012072/134264533-68e181db-bafb-42ba-8a6e-6c9502892070.png)

https://app.planetscale.com/binaryhands/leads/deploy-requests/1/diff

**Comment by jgreet** [comment, marked as answer]
Hi @Overload119 the error should have been visible on the summary page:

https://app.planetscale.com/binaryhands/leads/deploy-requests/1

Looks like you were able to get this deployed OK?

**Comment by Overload119** [comment]
Yup it worked eventually.
